### ISSUE TITLE
2024-huggingface-malicious-pickle-models

### ISSUE THREAD

**Issue by utopiazzr** [Narwhal-Lab/Narwhal-aicode-risks#47]
### Case title (English, concise) / 英文标题

Hugging Face Malicious Pickle Model Backdoors

### 中文标题(可选)

Hugging Face 恶意 Pickle 模型后门事件

### Submission bucket / 投稿类型

case (confirmed)

### Risk category / 风险类别

supply-chain — Hallucination & Supply Chain

### Severity / 严重度

high

### Severity basis / 严重度依据

cvss

### AI tools involved / 涉及的 AI 工具

  - "Hugging Face Hub"   - "PyTorch models"   - "TensorFlow Keras models"   - "Transformers model loading workflows"

### TL;DR (one sentence) / 一句话摘要

"Malicious Hugging Face model files used unsafe deserialization to open shells on model-loading hosts."

### References / 参考来源

  - title: "JFrog: Data Scientists Targeted by Malicious Hugging Face ML Models with Silent Backdoor"
    url: "https://jfrog.com/blog/data-scientists-targeted-by-malicious-hugging-face-ml-models-with-silent-backdoor/"
  - title: "The Hacker News: Over 100 Malicious AI/ML Models Found on Hugging Face Platform"
    url: "https://thehackernews.com/2024/03/over-100-malicious-aiml-models-found-on.html"
  - title: "NSFOCUS: AI Supply Chain Security - Hugging Face Malicious ML Models"
    url: "https://nsfocusglobal.com/ai-supply-chain-security-hugging-face-malicious-ml-models/"
  - title: "JFrog and Hugging Face Team to Improve Machine Learning Security and Transparency"
    url: "https://investors.jfrog.com/news/news-details/2025/JFrog-and-Hugging-Face-Team-to-Improve-Machine-Learning-Security-and-Transparency-for-Developers/default.aspx"
  - title: "Hugging Face: Safetensors"
    url: "https://github.com/huggingface/safetensors"

### AI-attribution evidence / AI 参与证据

[2024-huggingface-malicious-pickle-models.zip](https://github.com/user-attachments/files/29095350/2024-huggingface-malicious-pickle-models.zip)

### Details / 详情

传统软件供应链关注包管理器、安装脚本和构建产物。该事件说明，AI 模型本身也可以成为依赖项，模型文件和模型加载代码可携带执行语义。PyTorch Pickle、TensorFlow Keras Lambda layer 等格式在便利模型分发的同时，也可能让攻击者把反序列化变成代码执行入口。Hugging Face 已有 malware、pickle、secret scanning 和 safetensors 等机制，但公开证据显示，用户仍可能在风险提示下下载并执行不可信模型。

### Pre-flight self-check / 投稿前自查

- [x] I have personally opened each reference URL and confirmed it loads and says what I claim it says. / 我已逐条 fetch 过 URL 并确认内容与所声称的一致。
- [x] I have not conflated a controlled experiment / red-team test / hypothetical scenario with a real disclosed incident. / 我没有把研究 / 红队测试 / 假设场景包装成真实事件。
- [x] Loss / scope figures cited are from a primary source (or labelled clearly as estimates). / 引用的损失或规模数字来自一手来源(或明确标注为估计)。
- [x] I understand the repository is licensed under CC BY 4.0 and consent to my submission being licensed under the same terms. / 我理解本仓库使用 CC BY 4.0 协议,并同意以同样条款贡献内容。

### How would you like to be credited? / 您希望如何署名?

utopiazzr
